src/components/AnnouncementInfo/AnnouncementInfo.module.scss

```diff
@@ -0,0 +1,79 @@
+@import 'styles/uswdsDependencies';
+@import 'styles/sfds/sfdsDependencies';
+
+.announcementContainer {
+  display: flex;
+  align-items: center;
+  justify-content: space-evenly;
+  height: 100%;
+  padding-left: 4%;
+  padding-right: 4%;
+  width: auto;
+}
+
+.date {
+  font-family: 'Sharp Sans';
+  font-style: normal;
+  font-weight: 400 !important;
+  font-size: 18px;
+  line-height: 23px;
+  color: $black;
+}
+
+.gridContainer {
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  grid-template-rows: auto;
+  align-items: center;
+
+  p {
+    margin: 0;
+    grid-row: 2 / 2;
+    align-self: flex-start;
+    font-size: 1rem;
+  }
+
+  p + div {
+    grid-row: 1 / 3;
+    display: flex;
+    justify-content: center;
+  }
+
+  > div > a {
+    font-family: 'Trebuchet MS';
+    font-style: normal;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 19px;
+    text-align: center;
+    white-space: nowrap;
+  }
+}
+
+.title {
+  font-family: 'Trebuchet MS';
+  font-style: normal;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 150%;
+  color: $black;
```

Comment:
I would try to avoid adding so many specific type styles -- they should largely be applied already by higher level CSS. This will make it harder to change or overwrite as the design system evolves and when applying different color themes.